package-lock.json

```diff
@@ -1785,9 +1785,9 @@
       }
     },
     "node_modules/@babel/compat-data": {
```

Comment:
@alexanderleegs since there are no package.json changes are the lock file changes needed/intended?

Reply:
yup, it should be fine, removed in https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/68ec476d487f67d7bce2ba4e9731b80686c9d980!